patterns/pe.hexpat

```diff
@@ -104,9 +148,31 @@ struct DOSStub {
 	u8 data[11];
 };
 
+union SectionMisc {
+	u32 physicalAddress;
+	u32 virtualSize;
+};
+
+struct Section {
+	s8 name[8];
```

Comment:
Instead of `s8`, `char` can be used here. A `char` array will be displayed as string instead of an array in the pattern data view